assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -151,14 +183,29 @@ const contribCaptionRadios = {
   ],
 };
 
+const showAnnual = getQueryParameter('showAnnual', 'false');
```

Comment:
Once you get the test framework up and running would this be replaced with the variant from the Redux store?

Reply:
Yes